source/content/guides/autopilot/08-autopilot-faq.md

```diff
@@ -38,13 +38,13 @@ Not yet. [Autopilot](/guides/autopilot) is not compatible with [Build Tools](/gu
 ## What versions of Drush are supported by Autopilot?
 
 The version(s) of Drush that are supported by Autopilot for sites:
-- **Up to (not including) Drupal 9**: Drush 8
-- **Drupal 9 with Integrated Composer**: Any; Autopilot will use Composer instead of Drush.
-- **Drupal 9 without Integrated Composer**: Drush is not supported. Drupal 9 requires Drush 10 or higher, but Autopilot is not compatible with Drush 10.
+- **Up to (not including) the latest version of Drupal**: Drush 8
```

Comment:
This should still say "Up to (not including) Drupal 9: Drush 8"

Reply:
The block about Autopilot and and drush compatibility should probably not be changed (all changes reverted).  I would ask someone on the Autopilot team for input.

Reply:
reverted edits.  made a note to follow up w/autopilot team for future update.